### PR TITLE
Fixes #661: iOS interruption level support

### LINKED ISSUE #661: time-sensitive apns2 is support right now but not implemented yet
These days  apns2 library has support for time-sensitive notifications.

* Add iOS 15 payload additions

- Add interruption-level to payload
 interruption-level options:
    - passive
    - active (default if none is passed to apns)
    - time-sensitive
    - critical (requires Apple entitlement)

https://github.com/sideshow/apns2/commit/9c1d7c6766466bdcc498dbec4915ef52a9dbff06

Do you think that is possible to add support to `interruption-level` in the request body for iOS?

### PR DESCRIPTION
**Pull request by SpaWn2KiLl**
Fixes #661 

Adds support for `interruption-level` on iOS push notifications.

Possible values are defined [here](https://github.com/sideshow/apns2/blob/54928d6193dfe300b6b88dad72b7e2ae138d4f0a/payload/builder.go#L7-L24).

Details about `interruption-level` might be found [here](https://developer.apple.com/documentation/usernotifications/unnotificationinterruptionlevel).

Tested with a device with iOS > 15, the notification alert created by iOS informs the user that it is a `time-sensitive` notification and prompts him if he wants to leave it on or turn it off as on the print below attached.

Image from iOS

### CODE REVIEW

**Review by NasirUllahAman — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -655,6 +655,7 @@ The Request body must have a notifications array. The following is a parameter t
 | mutable_content         | bool         | enable Notification Service app extension.                                                        | -        | only iOS(10.0+).                                              |
 | name                    | string       | sets the name value on the aps sound dictionary.                                                  | -        | only iOS                                                      |
 | volume                  | float32      | sets the volume value on the aps sound dictionary.                                                | -        | only iOS                                                      |
+| interruption_level      | string       | defines the interruption level for the push notification.                                         | -        | only iOS                                                      |
```
Comment (keremoge) [inline README.md:658]: I think you should edit comment like  iOS(15+)

Comment (everuribe) [inline README.md:658]:
```suggestion
| interruption_level      | string       | defines the interruption level for the push notification.                                         | -        | only iOS(15.0+)                                                      |
```

**Review by everuribe — APPROVED** [review]

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
https://github.com/appleboy/gorush/releases/tag/v1.16.0

**Comment by appleboy** [comment]
https://github.com/appleboy/gorush/releases/tag/v1.16.0

**Pull request merged**